File: copy-file.py
import shutil
import os

source = r'c:\Users\DELL\Desktop\teranggo\Fullstack\terango\app\custom-delivery\index-new.tsx'
dest = r'c:\Users\DELL\Desktop\teranggo\Fullstack\terango\app\custom-delivery\index.tsx'

try:
    shutil.copy2(source, dest)
    print("✓ File replaced successfully!")
    print("  index.tsx has been updated with the new redesign")
except Exception as e:
    print(f"✗ Error: {e}")
    exit(1)

File: copy-file.js
const fs = require("fs");

const source =
  "c:\\Users\\DELL\\Desktop\\teranggo\\Fullstack\\terango\\app\\custom-delivery\\index-new.tsx";
const dest =
  "c:\\Users\\DELL\\Desktop\\teranggo\\Fullstack\\terango\\app\\custom-delivery\\index.tsx";

try {
  fs.copyFileSync(source, dest);
  console.log("✓ File replaced successfully");
  console.log("  index.tsx has been updated with the new design");
} catch (error) {
  console.error("✗ Error copying file:", error.message);
  process.exit(1);
}
